src/pages/Proposal.vue

```diff
@@ -177,13 +291,38 @@ export default defineComponent({
 				};
 			}
 		},
-	},
-	setup() {
-		const formState = reactive(ProposalStore());
-		return {
-			formState,
-			validateMessages: VALIDATE_MESSAGES,
-		};
+		async optInLsigToApp(lsig: LogicSigAccount) {
+			try {
+				if (typeof this.daoStore.dao_id === "undefined") {
+					console.log("appId not defined");
+					return;
+				}
+				if (typeof this.daoStore.govt_id === "undefined") {
+					console.log("govt_id not defined");
+					return;
+				}
+				// fund lsig
+				await fundAmount(
```

Comment:
We only fund  to account if this account not enough money right?

Reply:
Yes